### PR TITLE
no homo

### PR DESCRIPTION
**Pull request by Andrei199991**
image

approve fast
@pikulet 

image
image

### CODE REVIEW

File: Games/core/Game.js [hunk 1]
```diff
@@ -167,9 +167,7 @@ module.exports = class Game {
       "pregameWait",
       this.pregameWaitLength * 60 * 60 * 1000,
       () => {
-        this.sendAlert(
-          "Waited too long to start...This game will be closed in the next 30 seconds."
-        );
+        this.sendAlert("you are gay");
```
Comment (DrSharky) [inline Games/core/Game.js:170]: Why?

**Review by pikulet — APPROVED** [review]
lgtm

File: Games/core/Game.js (hunk 1 above)
Comment (pikulet) [inline Games/core/Game.js:170]: no fufu no merge

File: Games/core/Game.js [hunk 2]
```diff
@@ -167,9 +167,7 @@ module.exports = class Game {
       "pregameWait",
       this.pregameWaitLength * 60 * 60 * 1000,
       () => {
-        this.sendAlert(
-          "Waited too long to start...This game will be closed in the next 30 seconds."
-        );
+        this.sendAlert("you are gay :fufu:");
```
Comment (pikulet) [inline Games/core/Game.js:170]: 3 hell yeahs and this gets merged

**Pull request merged**